_data/events/Vechta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GDCR18 in Vechta",
+  "url": "https://doodle.com/poll/v3795putggfv68ab",
```

Comment:
I am not so fond of using doodle for this kind of purpose. Is there any chance we could use some other mean (i.e. meetup.com, eventbrite or something similar)?

Reply:
We have a deal! Thanks!